core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -492,20 +498,25 @@ def _convert_proba_to_unified_form(self, y_pred_proba):
     def score(self, X, y, metric=None, sample_weight=None, **kwargs):
         if metric is None:
             metric = self.eval_metric
+
         if metric.needs_pred:
             y_pred = self.predict(X=X, **kwargs)
+        elif metric.needs_quantile:
```

Comment:
`or` is cleaner.

Reply:
This will be fixed by merging with the above case.